### PR TITLE
Changed BYOND Link to Secure HTTPS

### PR DESCRIPTION
**Pull request by ghost**
HTTPS should be the standard by now.

### CODE REVIEW

**Comment by MrStonedOne** [comment]
Byond does not support https

**Comment by MrStonedOne** [comment]
If you notice, the site is served over https, but all links use http, so it just downgrades on next click.

**Comment by MrStonedOne** [comment]
Giving the user a https link will lead them into a false sense of security and they may not notice the conversion down to unsecure on the next click.

**Comment by ACCount12** [comment]
One day Lummox would properly implement HTTPS. Maybe. No harm in changing the link now.

**Comment by ACCount12** [comment]
Just change link to this then:
https://www.byond.com/download/
The downloads themselves are HTTPS if you follow this link.

**Comment by duncathan** [comment]
I am getting a serious sense of déjà vu here WTF

**Comment by MrStonedOne** [comment]
likely from #31854

**Comment by ike709** [comment]
@duncathan Must be a glitch in the Matrix.

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ hassle if you want to make any changes at all, so it's not recommended.)
 ## INSTALLATION
 
 First-time installation should be fairly straightforward. First, you'll need
-BYOND installed. You can get it from http://www.byond.com/. Once you've done
+BYOND installed. You can get it from https://www.byond.com/. Once you've done
```
Comment (ShizCalev) [inline README.md]: Change to https://www.byond.com/download/

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by gamedeviso** [comment]
@ShizCalev done

**Pull request merged**